src/Integration/ExceptionContextIntegration.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Yii\Sentry\Integration;
+
+use Sentry\Event;
+use Sentry\EventHint;
+use Sentry\Integration\IntegrationInterface;
+use Sentry\SentrySdk;
+use Sentry\State\Scope;
+
+class ExceptionContextIntegration implements IntegrationInterface
```

Comment:
Could you briefly describe what it does exactly?

Reply:
When happens exception, if it has mehtod `context()`, return value of this mehtod will pass as extra data to the sentry event.